src/fsd/0-app/index.css

```
@@ -676,6 +676,24 @@ input.item-quantity-input[type='number'] {
         transform: none;
     }
 }
+@keyframes accountDockMenuIn {
```

Stylelint (17.12.0)

[error] Expected keyframe name "accountDockMenuIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)